pheno_utils/pheno_loader.py

```diff
@@ -64,6 +66,7 @@ class PhenoLoader:
         valid_stage (bool): Whether to ensure that all research stages in the data are valid.
         flexible_field_search (bool): Whether to allow regex field search.
         errors (str): Whether to raise an error or issue a warning if missing data is encountered.
+        auto_translate (bool): Whether to automatically translate the data from coding to english.
```

Comment:
consider the following alternative:
instead of the word "translation", and a boolean value,
use something like `preferred_language="coding"` (or a better similar name of your choice) that accepts ["hebrew", "english", "coding"].
this can be readily extended to other languages if needed.
also, perhaps in the config we can set the default value for `preferred_language` (like other PhenoLoader args). maybe some customers will prefer an english default, and others a coding default.

but maybe you considered this and had a good reason to go for a boolean translation approach.

Reply:
Wonderful

Reply:
updated